demo/view-timeline/index.html

```diff
@@ -169,7 +169,7 @@
   <input type="checkbox" name="timeline-insets" id="timeline-insets">
   <label for="timeline-insets">Use inset 100px 200px</label>
 </body>
-<script src="../../dist/scroll-timeline.js"></script>
+<script src="../../dist-lite/scroll-timeline-lite.js"></script>
```

Comment:
this is just dist/scroll-timeline-lite.js now.

Reply:
Thanks - fixed.